src/components/EditActivityPage/EditActivityPage.js

```diff
@@ -34,66 +35,80 @@ const EditActivityPage = ({
 
   const activityId = get(editedActivity, 'id', id);
 
-  const asyncEditActivity = useCallback(
+  const intl = useIntl();
+
+  const editActivity = useCallback(
     async activitySubmited => {
       try {
         await onStartEditActivity(activityId, activitySubmited);
         history.push(ACTIVITY_PLAN_ROUTE);
       } catch (error) {
-        // eslint-disable-next-line no-console
-        console.log(`Something went wrong durring activity editing: ${error}`);
+        errorNotification(
```

Comment:
I think, that we shouldn't show details of errors to end user, so I would skip the `error`. 
Also, you could create own hook (so function which use intl and return object with errorNotification, which receive path to translation.
```
const { showErrorNotification } = useToast();
showErrorNotification('notification.error.activityError')
```
It would really reduce amount of code and follow rule `single responsibility`.